Ginger/GingerCoreNET/ActionsLib/ActSetVariableValue.cs

```diff
@@ -134,6 +136,57 @@ public override void Execute()
                 {
                     ((VariableDynamic)Var).ValueExpression = this.Value;
                 }
+                else if (Var.GetType() == typeof(VariableNumber))
+                {
+                    try
+                    {
+                        var varNumber = ((VariableNumber)Var);
```

Comment:
We have this validation also for UI right?  can we create a Validation method and call the same method here as well.